dts/bindings/radio_fem/skyworks,sky66407-11 .yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+description: |
+    This is a representation of the Skyworks SKY66407-11 Radio Front-End module.
+    It is supplementing the generic representation of FEM with settling time and
+    gain values appropriate for SKY66407-11 device.
+
+compatible: "skyworks,sky66407-11"
+
+include: generic-fem-two-ctrl-pins.yaml
+
+properties:
+    ctx-settle-time-us:
+        type: int
+        default: 23
```

Comment:
You should provide some explanation of the chosen default values, like it is done e.g. in [sky66112-11.yaml](https://github.com/nrfconnect/sdk-nrf/blob/main/dts/bindings/radio_fem/skyworks%2Csky66112-11.yaml).

Reply:
Updated the explanations.